FortranCon2021-stdlib/README.md

```diff
@@ -0,0 +1,40 @@
+# FortranCon2021-stdlib
+
+**Authors**:
+[Nathaniel Shaffer](https://github.com/nshaffer) (University of Rochester), 
+[Gabriel Brown](https://github.com/ghbrown) (University of Illinois Urbana-Champagne), 
+[Ondřej Čertik](https://github.com/certik) (Los Alamos National Laboratory),
+[William Clodius](https://githiub.com/wclodius2),
+[Robert Cohn](https://github.com/rscohn2) (Intel Corporation)
+[Milan Curcic](https://github.com/milancuric) (University of Miami),
+[Laurence Kedward](https://github.com/lkedward) (University of Bristol),
+[Sebastian Ehlert](https://github.com/awvwgk) (University of Bonn),
+[gareth-nx ???](https://github.com/gareth-nx),
+[Aman Godara](https://github.com/Aman-Godara),
+[Jing ???](https://github.com/Jim-215-Fisher),
+[Chetan Karwa](https://github.com/chetankarwa),
+[Arjen Markus](https://github.com/arjenmarkus),
+[Ian Giestas Pauli](https://github.com/14NGiestas) (São Paolo University),
+[Wileam Phan](https://github.com/wyphan) (University of Tennessee, Knoxville), 
+[Ivan Pribec](https://github.com/ivan-pi), (Technical University of Munich),
+[Yuichiro Sakamoto](https://github.com/sakamoti),
+[Harris Snyder](https://github.com/hsnyder), 
+[Jeremie Vandenplas](https://github.com/jvdp1) (Wageningen University & Research),
+[Evan Voyles](https://github.com/ejovo13) (Kalamazoo College),
+[Carlos Une](https://github.com/brocolis)
+
+
+**To be acknowledged in the presentation (following comments):**
+
+**Track**: Junior presentations (15 + 5 minutes discussion)
+
+**Presenter**: [Nathaniel Shaffer](https://github.com/nshaffer)
+
+**Title**: Fortran Standard Library
+
+**Abstract**:
+The Fortran standard library is now well into its second year of development.
+In this past year, stdlib has grown not only in its functionality but in its contributor base, with most of the new development coming from new contributors, including two Google Summer of Code students. 
+In addition to new capabilities in stdlib itself, there have also been several technical improvements to the documentation, workflow, and build system.
```

Comment:
```suggestion
For example, in addition to new capabilities in stdlib itself, there have also been several technical improvements to the documentation, and build system.
The full API documentation is generated by the FORD tool, and can be found online.
Furthermore, the Github 's Continuous Integration pipeline is used to test all procedures derived types and modules at all pull requests, to ensure cross-platform support.
```

Reply:
Thanks, Jeremie. I'll adjust the phrasing to my taste later, but these are nice suggestions.